swift/codegen/schema.yml

```diff
@@ -445,7 +446,7 @@ LookupExpr:
   _extends: Expr
   _children:
     base_expr: Expr
-  member: Decl
+  member: Decl?
```

Comment:
In case you wonder, I had an assertion firing on this :slightly_smiling_face:

Reply:
OOI which lookup expr didnt have a decl? 🤔